pending/tg.js

```diff
@@ -0,0 +1,777 @@
+settings.hitmarker.custom Larawan ng Hitmarker
+settings.hitmarker.custom.paste URL ng Hitmarker Image URL
+settings.onKillS.style.0 Default
+settings.onKillS.style.1 Bawasan ang Resolusyon
+settings.onKillS.style.2 Bawasan ang FOV
+settings.onKillS.style.3 Taasan ang FOV
+settings.serverb.header Server Browser
+settings.serverb.oldBrowser Legacy Browser
+settings.serverb.defaultRegionOnly Default na Rehiyon Lamang
+settings.gameprivacy.header Pagkapribado ng Gameplay
+settings.gameprivacy.streamer Mode ng Streamer
+settings.gameprivacy.anon Anonymous Mode
+settings.gameprivacy.hideGameTitle Itago ang Pamagat ng Laro
+settings.gameplay.header Gameplay
+settings.gameplay.sensitivityX X Sensitivity
+settings.gameplay.sensitivityY Y Sensitivity
+settings.gameplay.aimsensX Aim X Sensitivity
+settings.gameplay.aimsensY Aim Y Sensitivity
+settings.gameplay.challenge Mga Pagsubok at Hamon
+settings.gameplay.invert Baligtarin ang Y-Axis
+settings.gameplay.onkillS Sa Pagpatay
+settings.gameplay.scrollDir Direksyon sa Pag-iskroll
+settings.controller.header Controller
+settings.controller.sensitivityX X Sensitivity
+settings.controller.disableCons Huwag paganahin ang Mga Controller
+settings.controller.sensitivityY Y Sensitivity
+settings.controller.aimsensX Aim X Sensitivity
+settings.controller.aimsensY Aim Y Sensitivity
+settings.controller.deadZoneL DeadZone (Kaliwa)
+settings.controller.deadZoneR DeadZone (Kanan)
+settings.controller.triggerThres trigger Threshold
+settings.controller.vibration Vibrasyon o Pagyanig
+settings.controller.gradSpeed Unti-unting Bilis
+settings.controller.invert Baligtarin ang Y-Axis
+settings.mobile.header Mobile
+settings.mobile.sensitivityX X Sensitivity
+settings.mobile.sensitivityY Y Sensitivity
+settings.mobile.aimsensX Aim X Sensitivity
+settings.mobile.aimsensY Aim Y Sensitivity
+settings.mobile.vibration Panginginig ng boses
+settings.mobile.gradSpeed Unti-unting Bilis (?)
+settings.mobile.invert Baligtarin ang Y-Axis
+settings.audio.header Audyo o Tunog
+settings.audio.sound Bolyum o Tunog Pangkalahatan
+settings.audio.ambient Bolyum ng Ambiyansa
+settings.audio.voice Bolyum ng Boses
+settings.audio.weps Bolyum ng Tunog ng Armas
+settings.audio.player Bolyum ng Player
+settings.audio.skin Bolyum ng Manlalaro
+settings.audio.ui UI Bolyum ng UI
+settings.audio.asset Bolyum ng Asset
+settings.audio.dialogue Bolyum ng dialogo
+settings.viewmodel.header Tingnan ang Modelo
+settings.viewmodel.fov Field of View (Larangan ng Pagtingin)
+settings.viewmodel.fps FOV ng Armas
+settings.viewmodel.bobbing Pagtalbog ng Armas
+settings.viewmodel.leaning Nakasandal ang Armas
+settings.viewmodel.show.primary Ipakita ang Pangunahing Armas
+settings.viewmodel.show.secondary Ipakita ang Pandagdag Armas
+settings.viewmodel.roundArms Pa-bilog na braso
+settings.viewmodel.show.melee Ipakita ang Patalim
+settings.viewmodel.show.ads Itago ang armas kapag naka-ADs
+settings.viewmodel.toggleAim i-toggle ang pag-aim
```

Comment:
```suggestion
settings.viewmodel.toggleAim Pag-toggle ng pagpuntirya
```

Reply:
I'm not sure if most natives know "pagpupuntirya"

Reply:
> I'm not sure if most natives know "pagpupuntirya"

Hmm, I'll try to find a new word.. If not let's just stick with pag-aim or ADS